config/v2/config.go

```diff
@@ -428,6 +443,20 @@ func (c *Config) Generate(r *rand.Rand, size int) reflect.Value {
 		return nil
 	}
 
+	randKubernetesProvider := func(r *rand.Rand, s int) *KubernetesProvider {
+		if r.Float32() < 0.5 {
+			return &KubernetesProvider{}
+		}
+		return nil
+	}
+
+	randSentryProvider := func(r *rand.Rand, s int) *SentryProvider {
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`(*Config).Generate$14` - `s` is unused (unparam)